kahuna/public/stylesheets/main.css

```diff
@@ -518,6 +518,9 @@ input.search-query__input {
     display: none;
 }
 
+.file-uploader__select-files.button {
+    padding: 5px 10px !important; /* !important to override .button's !important */
```

Comment:
Not entirely sure. Git blame shows it was introduced [here](https://github.com/guardian/grid/commit/5c145fbe71a8999321f2e72482edabdfae552e77#diff-f7f0cf9f6b1bcaecc2be857deae091b2R67), but that's a reverting commit - will remove the rule and see how it changes things rather than going on a git history stack trace...